public/app/features/alerting/components/AlertingQueryPreview.tsx

```diff
@@ -21,18 +18,17 @@ const tabs = [
 ];
 
 interface Props {
-  queryRunner: PanelQueryRunner;
   getInstances: () => DataFrame[];
   queries: DataQuery[];
   onTest: () => void;
-  onRunQueries: () => void;
 }
 
-export const AlertingQueryPreview: FC<Props> = ({ getInstances, onRunQueries, onTest, queries, queryRunner }) => {
+export const AlertingQueryPreview: FC<Props> = ({ getInstances, onTest, queries }) => {
   const [activeTab, setActiveTab] = useState<string>(Tabs.Query);
-  const styles = getStyles(config.theme);
-  const observable = useMemo(() => queryRunner.getData({ withFieldConfig: true, withTransforms: true }), [queryRunner]);
-  const data = useObservable<PanelData>(observable);
+  const styles = useStyles(getStyles);
+
+  let data = {} as PanelData;
```

Comment:
do things not break? I think `PanelData` has mandatory properties
IMHO it would be more transparent and safer to make `data` optional on the `QueriesAndInstances` component then and preserve type safety as much as possible

Reply:
It's a bit unfortunate that we had to make changes here, this view won't work until we can actually run queries and display the result.